src/block/search/readme.md

```diff
@@ -8,32 +8,61 @@ This search block is used inside the site header mega block.
 	<section class="side-right row" id="search-modal">
 		<div class="column one">
 			<div class="header-search-input-wrapper">
-				<form method="get" action="https://search.wsu.edu/Default.aspx">
+				<form method="get" action="https://search.wsu.edu/Default.aspx" _lpchecked="1">
```

Comment:
Is the addition of the `_lpchecked="1"` attribute intentional? I'm guessing it's LastPass related...

Reply:
That's a whoops. :)